### PR TITLE
Add kramdown, emoji gems, and analytics to config

### PR DESCRIPTION
**Pull request by hectorsector**
The config in this repo is missing some key items. This PR adds these items in, namely:

- kramdown as the MD renderer
- various gems, including `jemoji`
- analytics

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,13 +2,62 @@
...
+
+
+# Site Settings
+locale                   : "en-US"
+# Keep original, do not replace
+title                    : "On Demand Training"
+title_separator          : "-"
+name                     : "GitHub, Inc."
+description              : "You're in control. We're with you along the way."
+url                      : "https://github.github.com" # the base hostname & protocol for your site e.g. "https://mmistakes.github.io"
```
Comment (crichID) [inline _config.yml]: @hectorsector is this still right? Should it be github.github.com? or services.github.com?

File: _config.yml [hunk 2]
```diff
@@ -2,13 +2,62 @@
...
+#url                      : ""
+baseurl                  : "/on-demand" # the subpath of your site, e.g. "/blog"
+gh_repo                  :
```
Comment (crichID) [inline _config.yml]: Can we use this to point to kit?

File: _config.yml (hunk 1 above)
Comment (hectorsector) [inline _config.yml]: I actually think we can remove this altogether.

File: _config.yml (hunk 2 above)
Comment (hectorsector) [inline _config.yml]: Possibly, let's do it in a separate PR though we can carefully explore what it does with that info, exactly.
